include/ua_plugin_nodestore.h

```diff
@@ -177,12 +180,23 @@ typedef enum {
         UA_DataSource dataSource;                                       \
     } value;
 
+#ifdef UA_ENABLE_HISTORIZING
+typedef struct {
+    UA_UInt64 historizingCallbackId;
+    UA_NodeId * nodeId;
+} UA_HistorizingInternal;
+#endif
+
 typedef struct {
     UA_NODE_BASEATTRIBUTES
     UA_NODE_VARIABLEATTRIBUTES
     UA_Byte accessLevel;
     UA_Double minimumSamplingInterval;
-    UA_Boolean historizing; /* currently unsupported */
+    UA_Boolean historizing;
```

Comment:
Can we have a callback in the global HDA plugin to query whether a node is historizing?